src/R/Editor/Impl/Signatures/SignatureHelpSource.cs

```diff
@@ -20,9 +21,11 @@
 namespace Microsoft.R.Editor.Signatures {
     sealed class SignatureHelpSource : ISignatureHelpSource {
         private ITextBuffer _textBuffer;
```

Comment:
Can't be readonly, it becomes `null` in `Dispose`. Do you want me to remove that part from `Dispose` method?

---
In reply to: [68842049](https://github.com/Microsoft/RTVS/pull/2022#discussion_r68842049) [](ancestors = 68842049)